addon/components/dropdown-link.hbs

```diff
@@ -0,0 +1,8 @@
+<LinkTo
+    @route={{@route}}
```

Comment:
we're missing a few args here, `@models`, `@activeClass`, `@query`, `@replace`, and `@current-when`.

see: https://github.com/Gavant/gavant-ember-button-link/blob/master/addon/components/button-link.hbs#L3

Note that for `@models` you'll probably want to create a getter in the component like I did in the button-link component, as ember expects an empty array if there are no models at all for the route. (getter will be needed for `activeClass` as well).